### PR TITLE
[REF] account_chatter: Show account.account chatter only for specific group

### PR DESCRIPTION
**Pull request by edy1192**
-  Show account.account chatter only for the group `group_show_account_chatter_notifications`.

### CODE REVIEW

**Comment by edy1192** [comment]
Hi @luisg123v could you check this please?

Regards.

File: account_chatter/views/account_account_views.xml [hunk 1]
```diff
@@ -6,6 +6,7 @@
         <field name="model">account.account</field>
         <field name="inherit_id" ref="account.view_account_form"/>
         <field name="groups_id" eval="[(4, ref('account_chatter.group_show_account_chatter_notifications'))]"/>
+        <field eval="100" name="priority"/>
```
Comment (luisg123v) [inline account_chatter/views/account_account_views.xml]: name goes first

File: account_chatter/views/account_account_views.xml [hunk 2]
```diff
@@ -16,5 +17,13 @@
             </xpath>
         </field>
     </record>
-
+    <record id="view_account_form_inherit_hidden" model="ir.ui.view">
```
Comment (luisg123v) [inline account_chatter/views/account_account_views.xml]: same ID as inherited view, the view with the group should be the one with a different ID

File: account_chatter/views/account_account_views.xml [hunk 3]
```diff
@@ -16,5 +17,13 @@
...
+        <field name="name">account.account.form.inherit.hidden</field>
```
Comment (luisg123v) [inline account_chatter/views/account_account_views.xml]:
```suggestion
        <field name="name">account.account.form.inherit.account.chatter</field>
```

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Comment by edy1192** [comment]
Done @luisg123v

File: account_chatter/views/account_account_views.xml [hunk 4]
```diff
@@ -16,5 +17,13 @@
             </xpath>
         </field>
     </record>
-
+    <record id="view_account_form" model="ir.ui.view">
+        <field name="name">account.account.form.inherit.account.chatter</field>
+        <field name="model">account.account</field>
+        <field name="inherit_id" ref="account.view_account_form"/>
+        <field eval="99" name="priority"/>
```
Comment (luisg123v) [inline account_chatter/views/account_account_views.xml]: You missed this one

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Review by luisg123v — APPROVED** [review]

**Pull request merged**